admin_site/system/views.py

```diff
@@ -1806,18 +1819,25 @@ def get_category(name):
                                           'doc/HOWTO_INSTALL_SERVER.txt'),
             'developer_guide': os.path.join(settings.SOURCE_DIR,
                                             'doc/DEVELOPMENT_HOWTO.txt'),
-            'release_notes': os.path.join(settings.SOURCE_DIR,
-                                          'NEWS'),
-            'create_bibos_image':os.path.join(tech_path,
-                             'HOWTOCreate_a_new_OS2borgerPC_image_from_scratch.txt')
-            ,
-            'save_harddisk_image':os.path.join(tech_path,
-                              'HOWTO_save_a_OS2borgerPC_harddisk_image.txt')
-            ,
-            'build_bibos_cd': os.path.join(tech_path,
-                'HOWTOBuild_OS2borgerPC_CD_from_clonezilla_image.md'
-            )
-            ,
+            'release_notes':
+                os.path.join(
+                    settings.SOURCE_DIR,
+                    'NEWS'
+                    ),
```

Comment:
Comment about placement of parantheses from above also applies here and on the following lines.